### ISSUE TITLE
Docs: Tricks to increase scan rate

### ISSUE THREAD

**Issue by boredsquirrel** [mjaakko/NeoStumbler#323]
### Description

This will need some tests to get data.

I would like to help get actions you can take, to increase the scanning of networks.

### Suggested implementation

A doc page here in the repo, to collect things like that.

### Additional information

My experience:
- keeping the app in the foreground
- keeing the display on (some OSses may need caffeine)
- keeping the phone charging on a powerbank, maybe fast charging maybe not
- disabling energy saver

There is this app [SaverTuner](https://codeberg.org/s1m/savertuner) that I use for energy saving. With ADB you can grant it access to "write secure settings".

This could be an optional ability of NeoStumbler too, to disable all energysavers kinda in the system. I dont know what could work here.

[I asked the SaverTuner dev about what they do](https://codeberg.org/s1m/savertuner/issues/57) and if they could help with  creating a "full power" mode.

Getting an app with such permissions into app stores may introduce friction. So some way to have an APK without that feature might be needed.

### New request

- [X] I have checked that a similar feature hasn't been requested yet

**Comment by mjaakko** [comment]
Good idea. I'm planning on creating an user guide for NeoStumbler as the amount of features increases and it's not clear for new or non-technical users to understand some of them. The user guide could include a section on how to increase scan performance

If you want to help with this, feel free create a PR. There's docs/ folder where you can add a Markdowk file

**Comment by boredsquirrel** [comment]
Great! I will do when on time.

Could start with what we know, leaving out everything that is not clear yet (hardware specific max scan rates, fancy hidden permissions etc)

**Comment by mjaakko** [comment]
This might be useful to mention in the documentation: https://dontkillmyapp.com/

**Comment by joelkoen** [comment]
I think it would be good to mention that you can disable Android's WiFi scan throttling in developer options.

I've been playing around with the "Force full GNSS measurements" setting (also in developer options) and I think it makes a noticeable difference with GPS accuracy.

**Comment by mjaakko** [comment]
I've started creating an [user guide](https://github.com/mjaakko/NeoStumbler/blob/master/docs/user_guide.md). If there's anything you want to add there, just create a PR :)
